dashboard/dashboard_config.yml

```diff
@@ -9,4 +9,6 @@ environment:
   # Comment out if not using public pretty-URL
   pretty_url: https://user.o6s.io/function
   # Comment out if not using public pretty-URL
-  query_pretty_url: 'true'
\ No newline at end of file
+  query_pretty_url: 'true'
+  # Cookie root domain, set is using OAuth, used to remove a user's cookie
+  cookie_root_domain: 'system.o6s.io'
```

Comment:
This is templated, so wouldn't this need to be changed in ofc-bootstrap rather than here?

Reply:
This needs to be synced here -> https://github.com/openfaas-incubator/ofc-bootstrap/blob/master/templates/dashboard_config.yml

Reply:
Yeh, it will need changing there. The ones in that yaml looked like "example" settings so I added there for future reference.